303-quarkus-vertx-sql/src/test/java/io/quarkus/qe/vertx/sql/test/resources/Db2Resource.java

```diff
@@ -49,7 +52,7 @@ private void defaultDb2Container(Map<String, String> config) {
 
     @Override
     public void stop() {
-        if (Objects.nonNull(db2Container)) db2Container.stop();
+        Optional.ofNullable(db2Container).ifPresent(GenericContainer::stop);
```

Comment:
which is the additional value?

Reply:
Having `if` statements in a single line is generally a bad practice.